src/frontend/cypress/e2e/Checkout.cy.ts

```diff
@@ -0,0 +1,32 @@
+describe('Checkout Flow', () => {
+  beforeEach(() => {
+    cy.visit(Cypress.env('baseUrl'));
+  });
+
+  it('should create an order with two items', () => {
+    cy.get('[data-cy="product-card"]').first().click();
```

Comment:
nit: could make a function like this to simplify the commands for reading
```javascript
const cyget(name) => cy.get(`[data-cy="${name}"]`)
```

Reply:
maybe also we can define all the `data-cy` values added to the frontend code in a file and pass Enum to this `cyget` method or similar so we can avoid potential typos. ex: 

```javascript
// before
cy.get('[data-cy="cart-item-count"]')

// after
import { CART_ITEM_COUNT } from './CypressFields'
cyget(CART_ITEM_COUNT)
```

just some ideas :)

Reply:
Great idea!